lms/templates/financial-assistance/financial_assessment_form.underscore

```diff
@@ -31,7 +31,7 @@
 			<div class="data"><%- name %></div>
 		</div>
 		<div class="info-column">
-			<div class="title"><%- gettext('Country of residence') %></div>
+			<div id="user-country-title" class="title"><%- gettext('Country of residence') %></div>
```

Comment:
Nitpick: I suppose it's just a matter of preference but I would use a class here instead of an ID to enable reusability (not that we would ever have 2 of these on the same page, it's more a coding convention).